WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -411,4 +436,22 @@ + (BOOL)reduceMotionEnabled
   }
   return NO;
 }
+
+/**
+ Returns defaul parameters for traversing elements tree from parents to children while requesting XCElementSnapshot.
+ The original maxDepth parameter in a dictionary provided by XCAXClient_iOS's method defaultParams is set to NSIntegerMax.
+ This method returns parameters with maxDepth set to FBDefaultSnapshotRequestMaxDepth
+*/
++ (NSMutableDictionary *)defaultSnapshotRequestParameters
+{
+  NSMutableDictionary *parameters = [NSMutableDictionary new];
+
+  parameters[@"maxArrayCount"] = @NSIntegerMax;
```

Comment:
I usually prefer `@{}` syntax

Reply:
Looks like ```@{}``` returns pointer of a type of a non mutable dictionary.

```
Incompatible pointer types initializing 'NSMutableDictionary *' with an expression of type 'NSDictionary *'
```

Reply:
[NSMutableDictionary dictionaryWithDictionary:@{...}]

Reply:
Fixed. done